website/i18n/en.json

```diff
@@ -104,6 +101,17 @@
       "integrations/tree-view-protocol": {
         "title": "Tree View Protocol"
       },
+      "lsp-extensions/debug-adapter-protocol": {
```

Comment:
These changes do not seem to be related, why is it needed?

Reply:
So it's not _needed_ per se, but when you run docusaurus locally it generates all the i18n stuff for the various pages. Last time when I was moving stuff around I must have not ran it again for these last two, so they were never created. So I didn't do anything to create these apart from just building the website.